addon/templates/components/frost-multi-select.hbs

```diff
@@ -17,7 +17,7 @@
     </div>
     <div class='multi-status'>
       <span class='number-selected'>{{selected.length}} selected</span>
-      <a href='#' class='clear' tabIndex={-1} onClick={{action "clearSelection"}}>Clear</a>
+      <a class='clear' tabIndex={-1} onClick={{action "clearSelection"}}>Clear</a>
```

Comment:
If config/environment.js uses `locationType: 'hash',`. This will reset them to the index